src/Service/IgnitionErrorListener.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Spatie\SymfonyIgnitionBundle\Service;
+
+use ob_start, ob_get_contents, ob_end_clean;
+use Spatie\Ignition\Ignition;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Event\ExceptionEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+class IgnitionErrorListener implements EventSubscriberInterface
+{
+    private $ignition;
```

Comment:
typed property?

Reply:
Good point -- and since this requires PHP 8.0 I changed it to use constructor property promotion.